javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -414,6 +419,19 @@ public String getValue() {
 		return Parameters.getParameterValue(this);
 	}
 
+	/**
+	 *
+	 * @param defaultValue
+	 * @return value of the parameter or the default value
+	 */
+	public String getValueOrDefault(String defaultValue) {

```

Comment:
`getValueOrDefault` is perhaps a good idea. But can we use is the `getValue` method for the moment?

Reply:
`getValueOrDefault` makes the code better readable without extra `if`s. Shall I remove it and use `getValue`?

Reply:
Yes, I think it's better for the moment.